### ISSUE TITLE
Please support Anonymized DNS in Adguard

### ISSUE THREAD

**Issue by nkoc6a15** [AdguardTeam/DnsLibs#120]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ x] I am running the latest version
- [x ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

**Comment by sfionov** [comment]
@nkoc6a15 Do you mean https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Anonymized-DNS ?

**Comment by nkoc6a15** [comment]
Yes. I see Adguard support DNSCrypt. So can you add support Anonymized DNS too? Maybe ODoH in the future when its stable?

**Comment by jamesta696** [comment]
Hi,
Is there any update on this issue?
Lots and of people waiting on this powerful feature.

Currently using Adguard - Quad9 with DNSCrypt and this feature would be a very important security update to it's platform.

It works perfectly when using Simple DNSCrypt for Windows but you don't get Adguard features.

If Adguard can implement this, it would be huge and you can also increase your userbase and sales.

We really need **Anonymized DNS Relays** for Adguard.
https://github.com/DNSCrypt/dnscrypt-resolvers/blob/master/v3/relays.md

**Comment by valantur** [comment]
somehow I thought this was supported out of the box.

**Comment by jamesta696** [comment]
> somehow I thought this was supported out of the box.

Adguard, out the box, does not support this feature. Although, they give you the option to choose which DNS service/protocol, they still haven't implemented the feature on Anonymized DNS Relays. 

To see this feature, you'd have to use SimpleDNSCrypt for Windows or InViziblePro on Android to achieve this concept.

SimpleDNSCrypt: https://github.com/instantsc/SimpleDnsCrypt
InViziblePro: https://github.com/Gedsh/InviZible

I've made a feature request on the Adguard Community Forums back in Feb 2022 but no reply or answer to my post.
https://forum.adguard.com/index.php?threads/please-implement-support-for-anonymized-dns-relays-when-using-dnscrypt-protocol.46817/

**Comment by Sn0whax** [comment]
Just wanted to jump in and say this would be amazing if implemented.

**Comment by jonsnow231** [comment]
> Just wanted to jump in and say this would be amazing if implemented.

Ya, I would prefer using open source software that did everything with a few clicks so I didn't have to set up the DNSCrypt Proxy with anonymized relays myself. Some free proprietary software like YogaDNS offers it.

**Comment by jamesta696** [comment]
2 years later after I filed this request 🥺
Adguard could seriously increase their revenue if they add powerful features like this.

**Comment by zubrRB** [comment]
I use Simple DNSCrypt, and in the DNS Protection of Adguard the indicated Systemic.

**Comment by jonsnow231** [comment]
> I use Simple DNSCrypt, and in the DNS Protection of Adguard the indicated Systemic.

Last I checked, [Simple DNSCrypt](https://github.com/bitbeans/SimpleDnsCrypt) didn't have anonymized dns.

**Comment by zubrRB** [comment]
In version 0.6.9 was added Anonymized DNS. https://github.com/bitbeans/SimpleDnsCrypt/releases

**Comment by jamesta696** [comment]
> Last I checked, [Simple DNSCrypt](https://github.com/bitbeans/SimpleDnsCrypt) didn't have anonymized dns. 

I assure you, the feature is Implemented and works, it's installed on my Win11 system and use it everyday as my default. There are many Anonymized DNS Relays to choose from 👍

**Comment by jamesta696** [comment]
> In version 0.6.9 was added Anonymized DNS. https://github.com/bitbeans/SimpleDnsCrypt/releases
> 

Correct, though the BitBeans version isn't maintained anymore, this fork has the latest DNSCrypt Proxy (v2.1.5) 

The fork by InstantSC isn't being maintained anymore as well.

This is the version I currently use, until Adguard can Implement the feature:
https://github.com/instantsc/SimpleDnsCrypt

**Comment by zubrRB** [comment]
> Correct, though the BitBeans version isn't maintained anymore, this fork has the latest DNSCrypt Proxy (v2.1.5)

To use Simple DNSCrypt with new versions of dnscrypt-proxy, need to remove the `demonize=false` in `dnscrypt-proxy.toml`.

> Last I checked, [Simple DNSCrypt](https://github.com/bitbeans/SimpleDnsCrypt) didn't have anonymized dns.

<details><summary>Anonymized DNS:</summary>

![1](https://github.com/AdguardTeam/DnsLibs/assets/50718960/8337cc4d-9b99-40b6-bcf8-ea29dca0e6ff)

![2](https://github.com/AdguardTeam/DnsLibs/assets/50718960/b972f966-4a97-4042-a9a0-63aa9c5f72af)

</details><br/>

**Comment by zubrRB** [comment]
> Correct, though the BitBeans version isn't maintained anymore, this fork has the latest DNSCrypt Proxy (v2.1.5)
> The fork by InstantSC isn't being maintained anymore as well.

I use this version Simple DNSCrypt: https://github.com/Dola-Shuvi/SimpleDnsCrypt/releases

**Comment by NSurtsev** [comment]
Still no support
